src/useCases/getOrCreateAccountId.ts

```diff
@@ -16,8 +16,14 @@ export const getOrCreateAccountId =
     const authId: string = session.user["id"];
     const accessToken = await getAccessToken();
 
-    return (
-      (await accountHolderApiGateway.getAccountHolderId(authId, accessToken)) ||
-      (await accountHolderApiGateway.createAccountHolderId(authId, accessToken))
+    const accountHolderId = await accountHolderApiGateway.getAccountHolderId(
+      authId,
+      accessToken
+    );
+    if (accountHolderId) return accountHolderId;
+
+    return await accountHolderApiGateway.createAccountHolderId(
+      authId,
+      accessToken
```

Comment:
Is this for readability?

The new assignment at line 19 and the `if` statement at line 23 adds a little imperative complexity.  What it replaces is (IMO) closer to idiomatic functional JS.   Probably a stye/preference thing - still happy to merge 👍🏻

Reply:
Readability. I think I discussed this with yourself and Issy, then the very first thing Matt commented on was the readability, so readability wins.